src/libaktualizr/package_manager/ostreemanager.h

```diff
@@ -65,6 +65,7 @@ class OstreeManager : public PackageManagerInterface {
 
  private:
   TargetStatus verifyTargetInternal(const Uptane::Target &target) const;
+  data::InstallationResult applyInstall(const Uptane::Target &target);
```

Comment:
Do we need this new method? We would have `install`, `completeInstall`, `finalizeInstall` and `applyInstall`. This is starting to get a bit confusing. Or maybe it's missing some docs and/or   comments?

Reply:
It's a private helper method and is not part of the class interface as the other listed methods are, but it, indeed is not really useful now and can be removed

Reply:
updated